src/components/DeploymentFlow/DeploymentFlowBox/DropdownMenu/DropdownMenu.component.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Menu, Dropdown } from 'antd';
+
+/**
+ * DeploymentFlowBox dropdown menu
+ */
+function DropdownMenu(props) {
+  const { disabled, onEdit, onRemove, children } = props;
+
+  const handleClick = (e) => {
+    if (e.key === 'edit') onEdit();
+    else if (e.key === 'remove') onRemove();
+  };
+
+  const menuOverlay = (
+    <Menu onClick={handleClick}>
+      <Menu.Item disabled={disabled} key='edit'>
+        Editar
+      </Menu.Item>
+      <Menu.Item disabled={disabled} key='remove'>
+        Remover
+      </Menu.Item>
+    </Menu>
+  );
+
+  return (
+    <Dropdown overlay={menuOverlay} trigger={['contextMenu']}>
+      {children}
+    </Dropdown>
+  );
+}
+
+DropdownMenu.propTypes = {
+  children: PropTypes.any,
+  disabled: PropTypes.bool,
+  onEdit: PropTypes.func.isRequired,
+  onRemove: PropTypes.func.isRequired,
+};
+
+DropdownMenu.defaultProps = {
+  experimentIsRunning: false,
+  interruptIsRunning: false,
+  status: '',
+  disabled: false,
+};
+
```

Comment:
@cedric-matheus uma dúvida:
este componente parece não ter nada de específico da tela de deployments. Ele não poderia estar em `src/components/DropdownMenu`?

No seu último PR notei que você tem colocado mais componentes dentro de subpastas (como o UserAvatar, por ex.).
https://github.com/platiagro/web-ui/tree/1e219603f59df0862e7fa1ba2377d629b1600672/src/components/TemplatesTable/UserAvatar
Não entendi porque. Meu entendimento era que a gente ia tentar reusar o máximo de componentes, mantendo-os fora das pastas específicas.

Reply:
Sobre a primeira pergunta:
Provavelmente vai ser isso que vai acontecer, mas atualmente, ele está sendo usado apenas por esse componente, então preferi deixar ele como subcomponente. Ele está bem isolado e bem simples de ser externalizado para quando for necessário.

Sobre a segunda pergunta:
A resposta é quase a mesma da primeira, esse aninhamento é apenas para simplificar o entendimento dos componentes, mas os mesmos estão bem desacoplados dos componentes "pais", sendo muito fácil de externalizá-los quando for preciso.